src/api/app/views/webui/requests/submissions/_breadcrumb_items.html.haml

```diff
@@ -0,0 +1,4 @@
+= render partial: 'webui/package/breadcrumb_items'
+
+%li.breadcrumb-item.active{ 'aria-current' => 'page' }
+  Submit
```

Comment:
To be consistent we should display `Submit Package` instead of only `Submit`

Reply:
Fixing